doc/static/css/common.css

```
@@ -226,7 +226,7 @@ div.numbered-step>h6::before {
     border: 1px solid #ddd;
     background-color: #fafafa;
     padding: 10px 10px 0px 0px;
-    margin: 5px;
+    margin: 25px;
```

Sorry, I was lazy - but 25px on all sides looks bad, we only want it on the left and on the bottom.
So:
```suggestion
    margin: 0px 0px 25px 25px;
```

---

fixed